plugin/pkg/global/resourcereferencemanager/admission_test.go

```diff
@@ -684,5 +684,239 @@ var _ = Describe("resourcereferencemanager", func() {
 				})))
 			})
 		})
+
+		Context("CloudProfile - Update Kubernetes versions", func() {
+			versions := []core.ExpirableVersion{
+				{Version: "1.17.2"},
+				{Version: "1.17.1"},
+				{Version: "1.17.0"},
+			}
+			shootOne := shootBase
+			shootOne.Name = "shoot-One"
+			shootOne.Spec.Provider.Type = "aws"
+			shootOne.Spec.Kubernetes.Version = "1.17.2"
```

Comment:
Can you add a CloudProfile name to Shoots and CloudProfile?
```suggestion
			shootOne.Spec.Kubernetes.Version = "1.17.2"
			shootOne.Spec.CloudProfileName = "aws"
```